go/oasis-test-runner/scenario/e2e/consensus_state_sync.go

```diff
@@ -86,7 +86,7 @@ func (sc *consensusStateSyncImpl) Run(childEnv *env.Env) error {
 
 	// Get the TLS public key from the validators.
 	var consensusNodes []string
-	for _, v := range sc.Net.Validators()[:2] {
+	for _, v := range sc.Net.Validators()[:3] {
```

Comment:
Instead of hardocing the `3` here, could change to "all but last" (and similarly bellow for the syncing validator could use "last"). That would also be in sync with the fixture where we append a validator, and would continue to work correctly in case the base fixture changes.

Reply:
Yeah, will do that and also add a log event assertion to make sure that state sync actually happened.